src/Microsoft.ML.Api/SchemaDefinition.cs

```diff
@@ -332,8 +332,18 @@ public static SchemaDefinition Create(Type userType)
 
                 if (fieldInfo.GetCustomAttribute<NoColumnAttribute>() != null)
                     continue;
-                var mappingAttr = fieldInfo.GetCustomAttribute<ColumnNameAttribute>();
-                var name = mappingAttr == null ? fieldInfo.Name : (mappingAttr.Name ?? fieldInfo.Name);
+                var mappingAttr = fieldInfo.GetCustomAttribute<ColumnAttribute>();
+                string attrName = null;
+                if (mappingAttr != null)
+                {
+                    attrName = mappingAttr.Name;
+                }
+                else
+                { 
+                    var mappingNameAttr = fieldInfo.GetCustomAttribute<ColumnNameAttribute>();
+                    attrName = mappingNameAttr != null ? mappingNameAttr.Name : null;
```

Comment:
>mappingNameAttr != null ? mappingNameAttr.Name : null; [](start = 31, length = 54)

This line is more clearly and concisely expressed as `attrName = mappingNameAttr?.Name;`